app/models/orphan.rb

```diff
@@ -69,18 +86,14 @@ class Orphan < ActiveRecord::Base
   accepts_nested_attributes_for :current_address, allow_destroy: true
   accepts_nested_attributes_for :original_address, allow_destroy: true
 
-  default_scope { includes(:partner, :orphan_sponsorship_status, :orphan_status, original_address: :province) }
+  default_scope { includes(:partner, original_address: :province) }
 
-  scope :active,
-        -> { joins(:orphan_status).
-            where(orphan_statuses: { name: 'Active' }) }
   scope :currently_unsponsored,
-        -> { joins(:orphan_sponsorship_status).
-            where(orphan_sponsorship_statuses: { name: ['Unsponsored', 'Previously Sponsored'] }) }
+    -> { where(sponsorship_status: [ Orphan.sponsorship_statuses[:unsponsored],
+                                     Orphan.sponsorship_statuses[:previously_sponsored] ]) }
   scope :high_priority, -> { where(priority: 'High') }
-  scope :deep_joins, -> { joins(:orphan_sponsorship_status).joins(:original_address).joins(:partner) }
```

Comment:
No longer need the `.deep_joins` scope since the default scope includes same tables.